### PR TITLE
v0.21 - Which Puppy

### PR DESCRIPTION
**Pull request by shiftkey**
- [x] release notes
- [x] version bump
- [x] update linqpad scripts to fix regressions
- [x] integration tests all pass
- [x] :thumbsup: from @Haacked or @ryangribble 
- [x] tag and publish to NuGet
- [x] :shipit:

### CODE REVIEW

**Comment by shiftkey** [comment]
@ryangribble @haacked @dampir any feedback on the release notes before I :ship: this tomorrow?

File: ReleaseNotes.md [hunk 1]
```diff
@@ -1,3 +1,44 @@
...
+ - `I(Observable)GitHubClient.Release`
+ - `I(Observable)GitHubClient.Notification`
+ - `I(Observable)GitHubClient.GitDatabase`
+ - `I(Observable)GitHubClient.SshKey`
+ - `I(Observable)GitHubClient.Repository.RepositoryComments`
+ - `I(Observable)GitHubClient.Repository.CommitStatus`
+ - `I(Observable)GitHubClient.Repository.RepoCollaborators`
+ - `I(Observable)GitHubClient.Repository.Commits`
+
+This method has also been removed, but is no longer supported through the API:
```
Comment (Haacked) [inline ReleaseNotes.md]: The `, but` here feels like the second clause contrasts with the first, but it really goes hand in hand with it. How about,

"This method is no longer supported through the API and has been removed from Octokit.net." or something like that?

**Comment by Haacked** [comment]
One minor nit, otherwise :+1: to the release notes.

**Comment by shiftkey** [comment]
This has shipped:

 - https://www.nuget.org/packages/Octokit/0.21.0
 - https://www.nuget.org/packages/Octokit.Reactive/0.21.0

**Comment by Haacked** [comment]
:party:

**Pull request merged**